syncer/test/.eslintrc.json

```diff
@@ -0,0 +1,13 @@
+{
```

Comment:
consolidate this config with `syncer/` or better yet, the to-be root config... there's too many configs!